.github/workflows/windows.yml

```
@@ -0,0 +1,59 @@
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [windows-latest]
+        build_type: [Release, Debug]
+        c_compiler: [cl]
+
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)